layouts/partials/pricing/faqs.html

```diff
@@ -8,49 +8,56 @@ <h2>FAQs</h2>
+            <summary class=" ">How does Budibase pricing work?</summary>
+            <p class="mb-2 text-sm mx-3  opacity-75">You can get started with Budibase today for free (cloud or self-host).</p>
+            <p class="mb-2 text-sm mx-3  opacity-75">
+                Once your team is ready to collaborate with more users, you can check out our Premium, Business and Enterprise plans based on your user need.
+                User pricing ranges from $8-$16 per user (per annum), with custom pricing available for Enterprise plans. 
+            </p>
         </details>
         <details class="py-3">
-            <summary class=" ">Does Budibase offer a discount for nonprofits?</summary>
-            <p class="mb-2 text-sm mx-3  opacity-75">Budibase is happy to offer eligible nonprofit organizations a discount on a Budibase Premium or Budibase Business annual plan. </p>
+            <summary class=" ">What counts as a Budibase user?</summary>
+            <p class="mb-2 text-sm mx-3  opacity-75">A Budibase user is anyone who requires the ability to login to Budibase in order to build, edit or use Budibase applications. 
+                This means that all users can not only be end users for applications, but also actively collaborate with others during the application building process.</p> 
         </details>
         <details class="py-3">
-            <summary class=" ">Can I self-host Budibase for free?</summary>
-            <p class="mb-2 text-sm mx-3  opacity-75">Yes, you can self-host Budibase for free. Check out our <a href="https://docs.budibase.com/docs/hosting-methods">self-host docs</a> for further guidance. </p>
+            <summary class=" ">Does Budibase offer discounts?</summary>
+            <p class="mb-2 text-sm mx-3  opacity-75">Budibase is happy to offer eligible nonprofit organizations a discount across all our paid tiers on annual plans. </p>
+            <p class="mb-2 text-sm mx-3  opacity-75">Volume based discounts are also available for Enterprise plans. <a href="http://localhost:1313/contact">Contact Sales</a> to find out more.</p>
         </details>
         <details class="py-3">
-            <summary class=" ">I've hit my user limit, can I add more users to my plan?</summary>
-            <p class="mb-2 text-sm mx-3  opacity-75">If you are on the free plan, you have to upgrade to one of the following plans; Premium, Business or Enterprise. If you are on a paid plan, you can add as many users as you please.</p>
+            <summary class=" ">What happens when I hit my limits?</summary>
+            <p class="mb-2 text-sm mx-3  opacity-75">If you hit the user limits on the free plan, you will need to upgrade to a paid plan. If you reach our data or automation limits, you’ll still be able to use Budibase. We’ll let you know about the overage, and give you a grace period to find a plan that fits your needs!
```

Comment:
The grace period isn't a thing right now, as soon as you hit your limits you'll not be able to use that particular feature. I can see you didn't add this, but now might be a good time to update to avoid confusion